### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by pgueckel** [web-scrobbler/web-scrobbler#3091] [Ideas]
1. I would like Web Scrobbler to be able to record listens from Naxos Music Library. It is a huge site with tens of thousands of albums, available for free through many public libraries and educational institutions and by paid membership.
2. I have a large (ripped) CD collection and would like to be able to manually enter my listens.
3. I only use ListenBrainz, not the other sites, for recording listens. All of my albums are catalogued in MusicBrainz. It would be really cool if I could just click on the album I am playing from one of my collections in my MusicBrainz account and have it automatically appear as 'playing now' in ListenBrainz.

**Comment by mountainash** [comment]
2. Unless you've uploaded your audio files to an online streaming service, it's probably better handled by some local (non-browser) based player/scrobbler (eg. NepTunes on macOS), but these are still often based on usage - manual entries is not really the point of Scrobbing software.

Probably better to break these (1 & 3) requests out into 2 different discussions.

**Comment by inverse** [comment]
@pgueckel For point 1 it's already in - See https://github.com/web-scrobbler/web-scrobbler/pull/3047

2. I agree with @mountainash it should be handled outside the scope of this extension.

3. Not sure I follow what you want? 

Normally these sorts of requests are handled through issues which Is why it probably slipped through people answering.

**Reply by pgueckel** [reply]
I realize that both #2 & #3 are out-of-scope.

Thanks for taking the time to read them, anyway :-)
